helm/alfresco-content-services/charts/alfresco-search/charts/alfresco-insight-zeppelin/templates/NOTES.txt

```diff
@@ -8,5 +7,4 @@ You can access Alfresco Insight Zeppelin using:
 If you have a specific DNS address for the cluster please run the following commands to get the application paths and configure ACS:
 
 helm upgrade --reuse-values {{ .Release.Name }} --set externalProtocol="http" --set externalHost="domain.com" --set externalPort="80" alfresco/alfresco-content-services
-{{ end }}
 {{- end }}
```

Comment:
does it make sense after removing the enabled logic? can we get rid of the entire file?

Reply:
Looks like that file actually never was included only the NOTES from the main charts are rendered. Will remove and maend the root NOTES

Reply:
and let's try ith a bit of fun :)